.github/workflows/e2e.yaml

```
@@ -79,3 +79,37 @@ jobs:
         curl -s -H "Content-Type: application/json" -d @e2e/data/input0.json http://localhost:8080/v2/models/my-inference-service2/infer | jq
         curl -s -H "Content-Type: application/json" -d @e2e/data/input1.json http://localhost:8080/v2/models/my-inference-service2/infer | jq
         curl -s -H "Content-Type: application/json" -d @e2e/data/input4.json http://localhost:8080/v2/models/my-inference-service2/infer | jq
+
+  e2e-python:
+    name: E2E using Python
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install UV
+      uses: astral-sh/setup-uv@v5
+    - name: Run tests
+      working-directory: python
+      run: |
+        make sync test
+    - name: Run the build and install the python wheel
+      working-directory: python
+      run: |
+        make build
+        pip install dist/modelcar_base_image-*.whl
```

Since we're installing uv just above, maybe we should use it here, or maybe add a comment if there is a reason we do want to stick with `pip`.
```suggestion
        uv pip install dist/modelcar_base_image-*.whl
```

---

I think the problem is that this suggested change will install it in the uv venv, while the intent here is to make the module available in between workflow steps